src/nvim/testdir/samples/memfile_test.c

```diff
@@ -72,7 +72,7 @@ test_mf_hash(void)
 	assert(mf_hash_find(&ht, key) == NULL);
 
 	/* allocate and add new item */
-	item = (mf_hashitem_T *)lalloc_clear(sizeof(*item), FALSE);
+	item = (mf_hashitem_T *)lalloc_clear(sizeof(*item), false);
```

Comment:
keep

Reply:
But that means we need to keep the define for `FALSE`?

Reply:
Not sure. Try removing `FALSE` but still keeping this `FALSE` and see if tests pass.